Description: Fix typos and macro issues in manpages
Author: Felix Lechner <felix.lechner@lease-up.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=915182
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=916946
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=962946
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=933773
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/md.4
+++ b/md.4
@@ -363,9 +363,9 @@ tab(;);
 ;Device #1;Device #2;Device #3;Device #4
 0x00;0;0;1;1
 0x01;2;2;3;3
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
 :;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
 0x80;254;254;255;255
 ;\\---------v---------/;\\---------v---------/
 ;RAID1;RAID1
@@ -392,9 +392,9 @@ C.
 ;Dev #1;Dev #2;Dev #3;Dev #4;Dev #5
 0x00;0;0;1;1;2
 0x01;2;3;3;4;4
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
 :;:;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.
 0x80;317;318;318;319;319
 ;
 .TE
@@ -454,15 +454,15 @@ C.
 ;
 0x00;0;1;2;3;\\ 
 0x01;4;5;6;7;> [#]
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 :;:;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 0x40;252;253;254;255;/
 0x41;3;0;1;2;\\ 
 0x42;7;4;5;6;> [#]~
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 :;:;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 0x80;255;252;253;254;/
 ;
 .TE
@@ -493,15 +493,15 @@ C.
 ;
 0x00;0;1;2;3;4;\\ 
 0x01;5;6;7;8;9;> [#]
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 :;:;:;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 0x40;315;316;317;318;319;/
 0x41;4;0;1;2;3;\\ 
 0x42;9;5;6;7;8;> [#]~
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 :;:;:;:;:;:;:
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;:
 0x80;319;315;316;317;318;/
 ;
 .TE
@@ -572,9 +572,9 @@ C.
 0x01;3;0;1;2;) AA~
 0x02;4;5;6;7;) AB
 0x03;7;4;5;6;) AB~
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
 :;:;:;:;:;  :
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
 0x79;251;252;253;254;) EX
 0x80;254;251;252;253;) EX~
 ;
@@ -605,9 +605,9 @@ C.
 0x01;4;0;1;2;3;) AA~
 0x02;5;6;7;8;9;) AB
 0x03;9;5;6;7;8;) AB~
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
 :;:;:;:;:;:;  :
-\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
+;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;\.\.\.;) \.\.\.
 0x79;314;315;316;317;318;) EX
 0x80;318;314;315;316;317;) EX~
 ;
@@ -987,7 +987,7 @@ other device(s).
 .I md
 cannot control the timeout that the underlying devices use to
 determine failure.  Any changes desired to that timeout must be set
-explictly on the underlying device, separately from using
+explicitly on the underlying device, separately from using
 .IR mdadm .
 
 If a FAILFAST request does fail, and if it is still safe to mark the
--- a/mdadm.8.in
+++ b/mdadm.8.in
@@ -461,11 +461,10 @@ number of spare devices.
 .BR \-z ", " \-\-size=
 Amount (in Kilobytes) of space to use from each drive in RAID levels 1/4/5/6.
 This must be a multiple of the chunk size, and must leave about 128Kb
-of space at the end of the drive for the RAID superblock.
-If this is not specified
-(as it normally is not) the smallest drive (or partition) sets the
-size, though if there is a variance among the drives of greater than 1%, a warning is
-issued.
+of space at the end of the drive for the RAID superblock. When specified as
+\(acmax\(ac (as it often is) the smallest drive (or partition) sets the size.
+In that case, a warning will follow if the drives, as a group, have sizes that
+differ by more than one percent.
 
 A suffix of 'K', 'M', 'G' or 'T' can be given to indicate Kilobytes,
 Megabytes, Gigabytes or Terabytes respectively.
@@ -682,7 +681,7 @@ A bug introduced in Linux 3.14 means tha
 started using a different layout.  This could lead to
 data corruption.  Since Linux 5.4 (and various stable releases that received
 backports), the kernel will not accept such an array unless
-a layout is explictly set.  It can be set to
+a layout is explicitly set.  It can be set to
 .RB ' original '
 or
 .RB ' alternate '.
@@ -924,6 +923,7 @@ the
 .B name
 will default to
 .IR home .
+(Does not work in Grow mode.)
 
 .TP
 .BR \-R ", " \-\-run
@@ -1445,7 +1445,7 @@ array, and the slot that it used is stil
 be added back to the array in the same position.  This will normally
 cause the data for that device to be recovered.  However based on the
 event count on the device, the recovery may only require sections that
-are flagged a write-intent bitmap to be recovered or may not require
+are flagged by a write-intent bitmap to be recovered or may not require
 any recovery at all.
 
 When used on an array that has no metadata (i.e. it was built with
@@ -2895,7 +2895,7 @@ long time.  A
 is required.  If the array is not simultaneously being grown or
 shrunk, so that the array size will remain the same - for example,
 reshaping a 3-drive RAID5 into a 4-drive RAID6 - the backup file will
-be used not just for a "cricital section" but throughout the reshape
+be used not just for a "critical section" but throughout the reshape
 operation, as described below under LAYOUT CHANGES.
 
 .SS CHUNK-SIZE AND LAYOUT CHANGES
@@ -2933,7 +2933,8 @@ option in Grow mode. Currently this work
 .B ppl
 and
 .B resync
-policies and allows to enable or disable the RAID5 Partial Parity Log (PPL).
+policies and allows one to enable or disable the RAID5 Partial Parity
+Log (PPL).
 
 .SH INCREMENTAL MODE
 
--- a/mdmon.8
+++ b/mdmon.8
@@ -115,7 +115,7 @@ container.  Some array management comman
 add are now only valid at the container level.  Attempts to perform
 these actions on member arrays are blocked with error messages like:
 .IP
-"mdadm: Cannot remove disks from a \'member\' array, perform this
+"mdadm: Cannot remove disks from a \(aqmember\(aq array, perform this
 operation on the parent container"
 .P
 Containers are identified in /proc/mdstat with a metadata version string
